dWorldServer/WorldServer.cpp

```diff
@@ -1244,7 +1245,11 @@ void HandlePacket(Packet* packet) {
 	}
 
 	default:
-		LOG("Unknown world packet received: %i", int(packet->data[3]));
+		if (packet->bitSize < (16 + sizeof(uint32_t) * 8)) break;
```

Comment:
This is already bounds checked far above in the same function if it gets this low